loops-and-arrays.js

```diff
@@ -31,17 +41,19 @@ const numbers = [2, 4, 5, 29, 38];
 
 
 
-
-// ==========================================
-// Opdracht 3
+/// Opdracht 3
 // Schrijf een script dat voor de maat van ieder vierkant in onderstaande array het volume uitrekent.
 // Vervolgens moet de huidige waarde in de array overschreven worden met: "Het volume van [x] is [y]"
 // Het volume is lengte x breedte x hoogte
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+console.log(squares)
 
+for (let i = 0; i < squares.length; i++) {
+    console.log("Het volume van " + squares[i] + " is " + (squares[i] ^ 3));
+}
```

Comment:
hier was het nog even een ingewikkeld sommetje zo, de uitkomsten zijn iets anders dan de verwachte uitkomsten, ^ lijkt niet helemaal te doen wat je ervan gedacht had in javaScript. In JavaScript gebruik je daar de operator ** voor,  maar omdat je door een list aan het itereren bent werkt dat niet, dus moet je het helemaal uitschrijven als squares[i] * squares[i] * squares[i]

Je kunt bij deze opdracht mooi je debugger inzetten op regel 55 om te zien wat er precies gebeurt als je daar mee gaat spelen, zodat je kunt zien wat JavaScript wel en niet doet met je code